app/views/shared/_navigation.html.erb

```diff
@@ -51,6 +51,27 @@
         </a>
       <% end %>
     <% end %>
+  <% else %>
+    <a class="nav-icon">
+      <img src="https://voices-of-consent-static-images.s3.amazonaws.com/Status.svg" title="Status">
```

Comment:
Having the full nav bar not appear if nobody is logged in might be a good thing. I kinda liked how your previous PR did that. Any opinions on this, @maebeale?